amr-wind/equation_systems/icns/source_terms/CMakeLists.txt

```diff
@@ -13,4 +13,5 @@ target_sources(${amr_wind_lib_name} PRIVATE
   ABLMesoForcingMom.cpp
   BurggrafFlowForcing.cpp
   RayleighDamping.cpp
+    NonLinearSGSTerm.cpp	
```

Comment:
unfortunately, clang-format adds these indents that we'd prefer not to have in the CMakeLists.txt files. It would be good to remove them. the formatting check only concerns with the .cpp and .H files

Reply:
A good way to do that automatically is this: `find . \( -name "*.cpp" -o -name "*.H" \) -exec clang-format -i {} +` (in the source directory)